clang/lib/CIR/Lowering/DirectToLLVM/LowerToLLVM.cpp

```diff
@@ -620,6 +684,28 @@ mlir::LogicalResult CIRToLLVMStoreOpLowering::matchAndRewrite(
   return mlir::LogicalResult::success();
 }
 
+/// Switches on the type of attribute and calls the appropriate conversion.
+mlir::Value lowerCirAttrAsValue(mlir::Operation *parentOp,
+                                const mlir::Attribute attr,
+                                mlir::ConversionPatternRewriter &rewriter,
+                                const mlir::TypeConverter *converter,
+                                mlir::DataLayout const &dataLayout) {
+  CIRAttrToValue valueConverter(parentOp, rewriter, converter);
+  auto value = valueConverter.visit(attr);
+  if (!value)
+    llvm_unreachable("unhandled attribute type");
```

Comment:
Can we call emitError here instead?

Reply:
We don't have access to `errorNYI` function, maybe we can add a MissingFeature for NYI implemented Attribute type in general